pasta_eln/GUI/form.py

```diff
@@ -48,8 +48,10 @@ def __init__(self, comm:Communicate, doc:dict[str,Any]):
                       if hasattr(self.comm.backend, 'configuration') else 300
       imageW, self.imageL = widgetAndLayout('V', mainL)
       Image(self.doc['image'], self.imageL, anyDimension=width)
-      imageW.setContextMenuPolicy(Qt.CustomContextMenu)
-      imageW.customContextMenuRequested.connect(lambda pos: initContextMenu(self, pos))
+      if '_id' in self.doc:
+        self.docID= doc['_id']  #required for hide to work
```

Comment:
Is this part of some other issue?

Reply:
Don't remember; but you can recreate the issue: open any measurement and the editor, and PASTA would write "widget has no docID" not causing any other big problems. This ensures that widget 'self' has a docID.